mappings/net/minecraft/particle/ParticleType.mapping

```diff
@@ -1,8 +1,9 @@
 CLASS net/minecraft/class_2396 net/minecraft/particle/ParticleType
-	FIELD field_11196 shouldAlwaysSpawn Z
+	FIELD field_11196 alwaysShow Z
```

Comment:
IIRC didnt this mean that the particle type should be shown _to other players_ other than the "current" one? Or was I mistaken here.

Reply:
No it's used to still display particles if you have particles disabled (attack indicator, campfire, sweep attack and explosions for instance)

Reply:
`required`?

Reply:
added a line of javadoc to make the param clearer